### PR TITLE
add scope argument to POD

### PR DESCRIPTION
**Pull request by nnutter**

### CODE REVIEW

**Comment by acoffman** [comment]
+1

**Comment by nnutter** [comment]
Test failure is external and will be addressed by genome/genome#459.

File: lib/perl/Genome/Sys/Lock.pm [hunk 1]
```diff
@@ -17,6 +17,10 @@ Keyword Arguments:
 C<resource_lock> is the path to the file representing the lock for some
 resource.
 
+=item scope
+
+C<scope> is the scope to which the lock is bound.  See C<scope()> for valid scopes.
```
Comment (tmooney) [inline lib/perl/Genome/Sys/Lock.pm]: Should this point to `scopes()` instead of `scope()`?

Comment (nnutter) [inline lib/perl/Genome/Sys/Lock.pm]: Fixed.

**Comment by tmooney** [comment]
+1 :penguin:

**Pull request merged**